ecosystem/sep-0018.md

```diff
@@ -40,16 +40,31 @@ start with a number.
 **Examples**
 
 ```
+app.example_org.id
 config.memo_required
-profile.name
-profile.language
-wallet.btc
-wallet.eth
+config.multisig.coordinator
 ```
 
 * **Term REGEXP:** `[_a-z][_a-z0-9]*`
 * **Key REGEXP:** `[_a-z][_a-z0-9]*(\.[_a-z][_a-z0-9]*)*`
 
+## Namespace Registry
+
+To guarantee interoperability and avoid naming conflicts, app-specific data
+must live under their reserved namespace:
+
+- `app`: Space dedicated to app-specific keys. Keys must be defined under
+  `app.{domain}`, with dot(s) replaced by underscores. For example,
+  "example.org" defines its keys under `app.example_org`.
+
+The other namespaces must be used according to their SEP documents. Whenever a
+SEP makes use of account data entries, it must be listed here:
+
+- `config`: Space dedicated to ecosystem-wide configuration. Those are values
+  that impact software logic.
+  - `config.memo_required`: [SEP29](sep-0029.md)
+  - `config.multisig`: [SEP19](sep-0019.md)
```

Comment:
I think the language of this section can be a bit softer as it was prior to this change. I've updated it to be similar to before, but retained the additional information that you're adding in this section.

```suggestion
The following namespaces have special meanings and their use should be reserved
for those meanings:

- `app.{domain}`: Namespace available to any domain operating an application and
  storing data entries. Data entry keys can be defined under `app.{domain}`, with
  dots in the domain replaced by underscores. For example, the website or app at
  domain `example.org` can define its data entry keys with namespace
  `app.example_org`.

- `config`: Namespace available for use by ecosystem-wide account configuration where data entry entries define behaviors or limitations of an account, for use in SEP standards. Known data entries in SEPs:
  - `config.memo_required`: [SEP29](sep-0029.md)
  - `config.multisig`: [SEP19](sep-0019.md)
```

Reply:
Haven't the words "must", "should" and "can" a precise meaning that must be respected when writing specifications?

Reply:
You're right that's a good point, it's better to not soften the language and use those terms. I removed the changing of those terms from my suggestion here. Is this fine with you? If so can you apply this suggestion and I'll merge the PR.
```suggestion
To guarantee interoperability and avoid naming conflicts, app-specific data
must live under their reserved namespace:

- `app`: Namespace available to app-specific keys. Keys must be defined under
  `app.{domain}`, with dot(s) replaced by underscores. For example,
  "example.org" defines its keys under `app.example_org`.

Other namespaces have special meaning according to their SEP documents.
When a SEP makes use of account data entries, it must be listed here and
referenced:

- `config`: Namespace available for use by ecosystem-wide configuration:
  - `config.memo_required`: [SEP29](sep-0029.md)
  - `config.multisig`: [SEP19](sep-0019.md)
```